Clients/GUI/Scripts/SignUper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+
+namespace GUI.Scripts
+{
+    public class SignUper
+    {
+        public static HttpStatusCode SignUp(SignUpViewModel insert)
```

Comment:
I heard that HttpStatusCode isn't  good method to return. It will be  return automatically. Probably, better to make it void?

Reply:
how do we know that something went wrong?